Normalizer/CustomEntityNormalizer.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Pim\Bundle\CustomEntityBundle\Normalizer;
+
+use Pim\Bundle\CustomEntityBundle\Entity\AbstractCustomEntity;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Internal API normalizer
+ *
+ * @author    JM Leroux <jean-marie.leroux@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class CustomEntityNormalizer implements NormalizerInterface
+{
+    /** @var array $supportedFormats */
+    protected $supportedFormats = ['internal_api'];
+
+    /** @var NormalizerInterface */
+    protected $pimSerializer;
+
+    /**
+     * @param NormalizerInterface $pimSerializer
+     */
+    public function __construct(NormalizerInterface $pimSerializer)
+    {
+        $this->pimSerializer = $pimSerializer;
```

Comment:
I really like the fact that you use `$pimSerializer` instead of just `$serializer` :+1: